cmd/launcher/options.go

```diff
@@ -107,6 +107,11 @@ func parseOptions(args []string) (*launcher.Options, error) {
 		}
 	}
 
+	// On windows, we should make sure osquerydPath ends in .exe
+	if runtime.GOOS == "windows" && !strings.HasSuffix(osquerydPath, ".exe") {
+		osquerydPath = osquerydPath + ".exe"
```

Comment:
Should we add some debug logging here? I _think_ this change could break some launcher invocations, if they were successfully running with suffix-less osqueryd binaries? Might be helpful to at least point them in the right direction.

Reply:
Ideally yes. But there's no logger instantiated at this point. :(

Reply:
ah that's a good point. I suppose the full osqueryd path _is_ logged out later on anyhow

Reply:
It's part of why I made the later logging lines more explicit. But yes, I'd love to log this. I guess future refactor can move that to main, and then log about it